code/datums/mutantraces.dm

```diff
@@ -1563,6 +1563,97 @@
 			src.real_name = "RELIQUARY TEST - DO NOT USE"
 			src.set_mutantrace(/datum/mutantrace/reliquary_soldier)
 
+/datum/mutantrace/cow
+	name = "cow"
+	// icon_state = "lizard"
```

Comment:
Probably remove?

Reply:
i already resolved that, also no fake cows allowed